sign.sh

```diff
@@ -2,7 +2,7 @@
 export "PATH=$PATH:/cygdrive/c/Program Files/Windows Kits/8.1/bin/x86"
 for f in Release Debug;
 do
-  signtool sign /f winsw_cert.pfx /t http://timestamp.verisign.com/scripts/timestamp.dll bin/$f/winsw.exe
+  signtool sign /f winsw-key.snk /t http://timestamp.verisign.com/scripts/timestamp.dll bin/$f/winsw.exe
```

Comment:
This sign script is not going to work well with the new project structure anyway. But it should not be required at all